Bottom: dbb0a04a22665199f0107d52feb614697a23d5e4
Top:    d8e306a0aacde0a259856fd058ee52324c8ce197
Author: scott snyder <sss@karma>
Date:   2020-02-12 11:10:41 -0500

Refresh of gitignore

---

diff --git a/.gitignore b/.gitignore
new file mode 100644
index 0000000..b25c15b
--- /dev/null
+++ b/.gitignore
@@ -0,0 +1 @@
+*~
